karma.conf.js

```diff
@@ -1,72 +1,73 @@
-'use strict';
```

Comment:
Did you accidentally format this as Windows line-endings? If that is the case, please revert it.